src/main/java/seedu/address/logic/commands/DeleteYCommand.java

```diff
@@ -89,6 +90,16 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         if (!(personToDelete == null)) {
+
+            if (!(model.hasExactPerson(personToDelete))) {
+                StaticContext.clearStaticContext();
+                throw new CommandException("Delete operation failed. If you modified the person (any field) "
+                        + "before pressing y or n, either\n"
+                        + "1. Undo the modification and try again, or\n"
```

Comment:
since we are reseting the delete operation, undoing the modification will still not work, also I think the word undo is quite ambiguous, might make them think theres a undo command

Reply:
Got it! Changing now!